src/main/java/uk/gov/justice/hmpps/offenderevents/config/ResourceServerConfiguration.java

```diff
@@ -53,7 +53,7 @@ public void configure(final HttpSecurity http) throws Exception {
                 .and().csrf().disable()
                 .authorizeRequests(auth ->
                         auth.antMatchers("/webjars/**", "/favicon.ico", "/csrf",
-                                "/health", "/info", "/ping", "/h2-console/**",
```

Comment:
Think you need to add `/health/ping`

Reply:
good spot, added wildcard so will match `/health` and everything below - https://docs.spring.io/spring-security/site/docs/5.3.0.RELEASE/api/org/springframework/security/web/util/matcher/AntPathRequestMatcher.html:

> Using a pattern value of /** or ** is treated as a universal match, which will match any request. Patterns which end with /** (and have no other wildcards) are optimized by using a substring match — a pattern of /aaa/** will match /aaa, /aaa/ and any sub-directories, such as /aaa/bbb/ccc.